paasta_tools/spark_tools.py

```diff
@@ -52,115 +48,10 @@ class SparkEnvironmentConfig(TypedDict):
 )
 
 
-@lru_cache(maxsize=1)
-def get_default_spark_configuration() -> Optional[SparkEnvironmentConfig]:
-    """
-    Read the globally distributed Spark configuration file and return the contents as a dictionary.
-
-    At the time this comment was written, the only bit of information that we care about from this file
-    is the default event log location on S3. See the TypedDict of the retval for the bits that we care about
-    """
-    try:
-        with open(DEFAULT_SPARK_RUN_CONFIG, mode="r") as f:
-            return yaml.safe_load(f.read())
-    except OSError:
-        log.error(
-            f"Unable to open {DEFAULT_SPARK_RUN_CONFIG} and get default configuration values!"
-        )
-    except yaml.YAMLError:
-        log.error(
-            f"Unable to parse {DEFAULT_SPARK_RUN_CONFIG} and get default configuration values!"
-        )
-
-    return None
-
-
 def get_webui_url(port: str) -> str:
     return f"http://{socket.getfqdn()}:{port}"
 
 
-def setup_event_log_configuration(spark_args: Dict[str, str]) -> Dict[str, str]:
```

Comment:
Whereever you have deleted code sections, can you add 1 liner comments examples:
* Moved to utils for reason xyz
Or * Deleted because it is deprecated now